src/main/webapp/js/issues-detail.js

```diff
@@ -166,7 +89,7 @@
          */
         if ($('#health-trend-chart').length) {
             view.getHealthTrend(isBuildOnXAxis, function (lineModel) {
-                $('#health-trend-chart').renderTrendChart(lineModel.responseJSON, redrawTrendCharts);
+                renderZoomableTrendChart('health-trend-chart', lineModel.responseJSON, redrawTrendCharts);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: ['renderZoomableTrendChart' is not defined. (no-undef)](https://app.codacy.com/app/uhafner/warnings-ng-plugin/pullRequest?prid=4437947)